<!doctype html>
<html lang="en">

<head>
    <title>Proteomics</title>
    <!-- Required meta tags -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <!--CSS-->
    <link rel="stylesheet" href="css/style.css">
    <link rel="stylesheet" href="css/proteomics.css">
    <!--Bootstrap-->
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet" 
    integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">
    <!--Icons-->
    <link rel="stylesheet" 
    href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.3/font/bootstrap-icons.min.css">
    <link rel="icon" href="img/Icon/favicon.ico">

</head>

<body>

<!--Nav Start-->
    <nav class="navbar navbar-expand-lg bg-body-tertiary fixed-top">
        <div class="container-fluid">
            <a class="navbar-brand" href="index.html"><img src="img/NBCC_Logo_Primary.png" alt="NBCC" width="30px"></a>
            <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
                <span>
                    <i class="bi bi-list"></i>
                </span>
          </button>
        <div class="collapse navbar-collapse" id="navbarSupportedContent">
        <!-- Left links -->
            <ul class="navbar-nav mb-2 mb-lg-0 me-auto">
        <!--Start of dropdown-->
                <li class="nav-item dropdown">
                    <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
                        Facilities
                    </a>
                    <ul class="dropdown-menu">
                        <li>
                            <a class="dropdown-item" href="proteomics.html">Proteomics</a>
                        </li>
                        <li>
                            <a class="dropdown-item" href="Next-Generation_Sequencing.html">Next-Generation Sequencing</a>
                        </li>
                        <li>
                            <a class="dropdown-item" href="High-Throughput_Screening.html">High-Throughput Screening</a>
                        </li>
                        <li>
                            <a class="dropdown-item" href="High-Content_Screening.html">High-Content Screening</a>
                        </li>
                        <li>
                            <a class="dropdown-item" href="Advanced_Imaging.html">Advanced Imaging</a>
                        </li>
                        <li>
                            <a class="dropdown-item" href="Flow_Cytometry.html">Flow Cytometry</a>
                        </li>
                    </ul>
                </li>
        <!--End of dropdown-->
                <li class="nav-item">
                    <a class="nav-link" href="resources.html">Resources</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link" href="publications.html">Publications</a>
                </li>
            </ul>
            <!--Right Nav-->
            <ul class="navbar-nav nav ms-auto">
                <li class="nav-item">
                    <a class="nav-link" href="news.html">News</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link" href="about.html">About</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link" href="team.html">Team</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link" href="contact.html">Contact</a>
                </li>
              <!--Icons-->
                <li class="nav-icon me-3 me-lg-0">
                    <a href="https://x.com/nbcc_ltri" class="nav-link">
                        <i class="bi bi-twitter-x"></i>
                    </a>
                </li>
                <li class="nav-icon me-3 me-lg-0">
                    <a href="https://www.linkedin.com/company/network-biology-collaborative-centre/" class="nav-link">
                    <i class="bi bi-linkedin"></i>
                    </a>
                </li>
            </ul>
          </div>
        </div>
    </nav>
<!--Nav End-->

    <!-- Main Header -->
    <header>
        <h1 class="caption">
            Proteomics
        </h1>
    </header>
    <!-- Main Header -->

    <!-- Hero Section -->
    <div class="hero-section">
        <div class="container">
            <div class="row">
                <div class="col-md-6 col-sm-12">
                    <h3>
                        An Introduction
                    </h3>
                    <p>
                        The NBCC provides full support for your proteomics needs from initial study concept to manuscript
                        preparation and data submission to public repositories. The facility uses bottom-up proteomics
                        (shotgun sequencing) to identify peptides using mass spectrometry and then matches them back to known proteins.
                        The facility has developed and validated a first-in-class pipeline for protein-protein interaction studies
                        that culminates in publication-ready visualizations. In addition, we provide advanced instrumentation and in-depth
                        expertise for differential proteome abundance and chemical proteomics studies.    
                    </p>
                    
                </div>
                <div class="col-md-6 col-sm-12">
                    <div class="">
                    <img src="img/Brendon Seale installing Flex source.jpg" alt="Brendon Installing Flex Source" width="100%">
                    </div>
                </div>
                <hr class="mt-2">
            </div>
        </div>
    </div>
    <!-- Hero Section -->

    <!-- Quick Nav -->
     <div class="quicknav">
        <h4 style="text-align: center;">
            Quick Navigation
        </h4>
        <span class="q-nav">
            <button>
                <a href="#first-section">
                    Services
                </a>
            </button>
            <button>
                <a href="#third-section">
                    Equipment
                </a>
            </button>
            <button>
                <a href="#second-section">
                    Pricing
                </a>
            </button>
            <button>
                <a href="#fourth-section">
                    Protocols
                </a>
            </button>
            <button>
                <a href="#protocol">
                    Policies
                </a>
            </button>
        </span>
    </div>
    <!-- Quick Nav -->

    <div id="first-section">
       
        <!-- Header 1 -->
        <div class="stretch-header pt-2 mt-3">
            <h1 class>
                Services
            </h1>
        </div>
        <!-- Header 1 -->

        <!-- IMG Section 1-->
        <div class="container mt-3">
            <div class="row">
                <div class="col-md-4 col-sm-12">
                    <div class="">
                        <img src="img/DDAvsDIA.svg" alt="DDAvsDIA" width="100%" id="myImg">
                        <!-- The Modal -->
                        <div id="myModal" class="modal">

                            <!-- The Close Button -->
                            <span class="close">&times;</span>
                        
                            <!-- Modal Content (The Image) -->
                            <img class="modal-content" id="myImg" src="img/DDAvsDIA.svg">
                        
                            <!-- Modal Caption (Image Text) -->
                            <div id="caption"></div>
                            <script>
                                // Get the modal
                                    var modal = document.getElementById("myModal");

                                    // Get the image and insert it inside the modal - use its "alt" text as a caption
                                    var img = document.getElementById("myImg");
                                    var modalImg = document.getElementById("img01");
                                    var captionText = document.getElementById("caption");
                                    img.onclick = function(){
                                    modal.style.display = "block";
                                    modalImg.src = this.src;
                                    captionText.innerHTML = this.alt;
                                    }

                                    // Get the <span> element that closes the modal
                                    var span = document.getElementsByClassName("close")[0];

                                    // When the user clicks on <span> (x), close the modal
                                    span.onclick = function() {
                                    modal.style.display = "none";
                                    }
                            </script>
                        </div>
                    </div>
                </div>
                <div class="col-md-8 col-sm-12">
                    <h3>
                        1. Mass Spectrometry Acquisition 
                    </h3>
                    <p>
                        We perform 2 main modes of mass spectrometry acquisition, but also offer labelled and targeted quantitative analysis. Users can submit already processed peptide solutions and opt for one of the following acquisition modes:
                    </p>
                </div>
            </div>
            <hr>
        </div>
        <!-- IMG Section 1-->
        
        <!-- Carousel -->
        <div class="container">
            <div class="accordion" id="accordionExample">
                <div class="accordion-item">
                <h2 class="accordion-header">
                    <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
                        Data-Dependent Acquisition (DDA) 
                    </button>
                </h2>
                <div id="collapseOne" class="accordion-collapse collapse show" data-bs-parent="#accordionExample">
                    <div class="accordion-body">
                        Data-dependent acquisition (DDA) fragments the top N (typically 10) most abundant precursors in each MS1 scan window. This allows for a confident matching
                        of fragmentation pattern (to determine peptide sequence) to the m/z (mass over charge). The MS1 scan detects all prescursors/peptides present and reports
                        their mass and charge. DDA results are easy to analyze but are often biased and stochastic because it selects and fragments the most abundant precursors.
                        DDA is suggested for simple mixtures (100 proteins or less), complex post-translation modifications studies, and de novo sequencing (not offered by the facility).
                        DDA is also our standard for protein-protein interaction (PPI) studies. DDA results through our facility are reported as spectral counts (number of times
                        a fragmentation pattern is matched to a peptide that is matched to a protein).  
                    </div>
                </div>
                </div>
                <div class="accordion-item">
                <h2 class="accordion-header">
                    <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
                        Data-Independent Acquisition (DIA)  
                    </button>
                </h2>
                <div id="collapseTwo" class="accordion-collapse collapse" data-bs-parent="#accordionExample">
                    <div class="accordion-body">
                        Data-independent acquisition (DIA) is an unbiased approach which breaks up the MS1 scan window into smaller windows of sizes 50 m/z or below. All the precursors
                        in each of these small windows are fragmented together. This produces a complex spectra which may contain fragmentation patterns from multiple precursors.
                        DIA data needs to be analyzed with complex computational algorithms that require more computational power than DDA. DIA is recommended for complex protein
                        mixtures (5000+ proteins) and simple post-translational modifications (phosphorylation or ubiquitin remnant after trypsin cleavage (GG)).
                    </div>
                </div>
                </div>
                <div class="accordion-item">
                <h2 class="accordion-header">
                    <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseThree" aria-expanded="false" aria-controls="collapseThree">
                        Labelled Quantitative Analysis   
                    </button>
                </h2>
                <div id="collapseThree" class="accordion-collapse collapse" data-bs-parent="#accordionExample">
                    <div class="accordion-body">
                        We offer services to label peptide samples with 
                        <a href=
                        "https://www.thermofisher.com/ca/en/home/life-science/protein-biology/protein-mass-spectrometry-analysis/protein-quantitation-mass-spectrometry/tandem-mass-tag-systems.html?cid=bid_pca_msp_r01_co_cp1359_pjt0000_bid00000_0se_gaw_bt_pur_con&ef_id=CjwKCAjwkJm0BhBxEiwAwT1AXNASmPL8oNXhmoXVMea9yzHRUs9PyTY3hJN3xyJkvDtESm41BJFgOBoCesYQAvD_BwE%3AG%3As&gad_source=1&s_kwcid=AL%213652%213%21597124685923%21e%21%21g%21%21tmt+thermo%2112451261088%21120049585642">tandem mass tags
                        </a>
                        (TMT) to multiplex samples for quantification, up to 18-plex, in one mass spectrometry acquisition run. TMT multiplexed samples will be analyzed
                        on our Thermo mass spectrometers. We also offer services to quantify samples labelled with other modalities such as SILAC and iTRAQ. Please contact
                        us to discuss your project and the most appropriate methodology.
                    </div>
                </div>
                </div>
                <div class="accordion-item">
                    <h2 class="accordion-header">
                    <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseFour" aria-expanded="false" aria-controls="collapseFour">
                        Targeted Quantitative Analysis (parallel reaction monitoring)
                    </button>
                    </h2>
                    <div id="collapseFour" class="accordion-collapse collapse" data-bs-parent="#accordionExample">
                    <div class="accordion-body">
                        For quantification of a specific set of proteins, peptides or post-translational modifications, we offer targeted quantification analysis on the Fusion
                        Tribrid Lumos through 
                        <a href=
                        "https://www.thermofisher.com/ca/en/home/clinical/clinical-translational-research/clinical-translational-research-learning-center/translational-research-information/proteomics-research-information/proteomics-protein-quantitation-information.html">parallel reaction monitoring
                        </a> 
                        (PRM). The costing will be determined after the project is defined and will include all assay setup costs.
                    </div>
                    </div>
                </div>
            </div>
            <hr>
        </div>
        
        <!-- Carousel -->

        <!-- IMG Section 2 -->
        <div class="hero-section">
            <div class="container">
                <div class="row">
                    <div class="col-md-8 col-sm-12">
                        <h3>
                            2. Protein-Protein Interaction (PPI) or Proximal Protein Mapping 
                        </h3>
                        <p>
                            We are specialists in protein-protein interaction (PPI) mapping. We offer reagents, expertise, and a data analysis pipeline to identify interacting
                            proteins (immunoaffinity purification (e.g., FLAG tag)) and proximal proteins (proximity-dependent biotinylation (PDB) methods 
                            (often referred to as BioID)). These assays can be performed re-iteratively for proteins in the same pathway or protein family to create maps of
                            signaling networks. See <a href="resources.html">resources</a> for reagents and <a href="#fourth-section">protocols</a> for sample preparation.
     
                        </p>
                        
                    </div>
                    <div class="col-md-4 col-sm-12">
                        <div class="">
                        <img src="img/PPI.jpg" alt="Protein-protein interaction" width="100%" id="myImg2">
                            </div>
                        </div>
                    </div>
                    <hr class="mt-2">
                </div>
            </div>
        </div>
        <!-- IMG Section 2 -->

        <!-- IMG Section 3-->
        <div class="container mt-3">
            <div class="row">
                <div class="col-md-4 col-sm-12">
                    <div class="">
                        <img src="img/Differential_proteome_abundance.svg" alt="DPA" width="100%" id="myImg3">
                        <!-- The Modal -->
                        <div id="myModal" class="modal">

                            <!-- The Close Button -->
                            <span class="close">&times;</span>
                        
                            <!-- Modal Content (The Image) -->
                            <img class="modal-content" id="img03">
                        
                            <!-- Modal Caption (Image Text) -->
                            <div id="caption"></div>
                            <script>
                                // Get the modal
                                    var modal = document.getElementById("myModal");

                                    // Get the image and insert it inside the modal - use its "alt" text as a caption
                                    var img = document.getElementById("myImg3");
                                    var modalImg = document.getElementById("img03");
                                    var captionText = document.getElementById("caption");
                                    img.onclick = function(){
                                    modal.style.display = "block";
                                    modalImg.src = this.src;
                                    captionText.innerHTML = this.alt;
                                    }

                                    // Get the <span> element that closes the modal
                                    var span = document.getElementsByClassName("close")[0];

                                    // When the user clicks on <span> (x), close the modal
                                    span.onclick = function() {
                                    modal.style.display = "none";
                                    }
                            </script>
                    </div>
                </div>
                </div>
                <div class="col-md-8 col-sm-12">
                    <h3>
                        3. Differential Proteome Abundance  
                    </h3>
                    <p>
                        We perform sample processing and global proteome analysis to determine differences between cell states. We comprehensively track changes between different
                        populations (e.g., normal vs disease) or treatments. To achieve high protein coverage, we identify and quantify proteins from these complex mixtures in an
                        unbiased manner on our state-of-the-art instruments (e.g., timsTOF Pro 2 or Orbitrap Astral).
                    </p>
                </div>
            </div>
            <hr>
        </div>
        <!-- IMG Section 3-->

        <!-- IMG Section 4 -->
        <div class="hero-section">
            <div class="container">
                <div class="row">
                    <div class="col-md-8 col-sm-12">
                        <h3>
                            4. Chemical Proteomics  
                        </h3>
                        <p>
                            Assessing target engagement is a key step in the drug development process. We offer pull-downs and competition assays to identify which proteins bind
                            small molecules or biologics. These tests enable the rapid assessment of on- and off-target binding during early-stage drug validation.       
                        </p>
                        
                    </div>
                    <div class="col-md-4 col-sm-12">
                        <div class="">
                        <img src="img/chem_proteomics.jpg" alt="Chem Proteomics" width="100%">
                        </div>
                    </div>
                    <hr class="mt-2">
                </div>
            </div>
        </div>
        <!-- IMG Section 4 -->

        <!-- IMG Section 5 -->
        <div class="hero-section">
            <div class="container">
                <div class="row">
                    <div class="col-md-3 col-sm-12">
                        <img src="img/DataAnalysis.jpg" alt="Data Analysis" width="100%">
                    </div>
                    <div class="col-md-9 col-sm-12">
                        <h3>
                            5. Data Analysis and Visualization 
                        </h3>
                        <p>
                            We have created resources to help manage, analyze, and visualize protein-protein interaction data. For protein-protein interaction (PPI) studies, we have a
                            complete pipeline going from storage and management of data with ProHits, searching of data with MSFragger, analysis of PPI data with SAINTexpress, and
                            visualization with ProHits-viz.  
                        </p>
                    </div>
                </div>
                
                <a href="https://prohitsms.com/Prohits_download/list.php">
                    <b>ProHits</b> 
                </a>
                <p>
                    ProHits is a laboratory information management system (LIMS) that manages and annotates data. 
                    <a href="https://www.nature.com/articles/nmeth.4256">
                        MSFragger
                    </a> 
                    and 
                    <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4102138/">
                        SAINTexpress
                    </a>
                     are integrated for a complete analysis pipeline.  
                </p>
                <a href="https://prohits-viz.org/">
                    <b>ProHits-viz</b> 
                </a>
                <p>
                    ProHits-viz is an online visualization tool for the seamless generation of heatmaps and dot plots from SAINTexpress results. Results can be exported for
                    Cytoscape interaction mapping.  
                </p>
                <a href="https://cell-map.org/">
                    <b>Cell-Map</b> 
                </a>
                <p>
                    Cell-Map is a resource of proximity-dependent biotinylation data from 192 subcellular markers in HEK293 cells. This is an online tool that can be used for
                    localization analysis of proximity-dependent biotinylation data.  
                </p>
            </div>
        </div>
    </div>

    <div id="third-section">

        <!-- Header 3 -->
        <div class="stretch-header pt-2 mt-3">
            <h1 class>
                Equipment
            </h1>
        </div>
        
        <!-- Header 3 -->
         <div class="container mt-3">
            <p>
                The NBCC provides proteomics services using 5 mass spectrometers from 3 different vendors: ThermoFisher Scientific, Bruker, and Sciex. All instruments can perform
                DDA and DIA.  
            </p>
            <div class="row">
                <div class="col-md-8 col-sm-12">
                    <h3>
                ThermoFisher Scientific
            </h3>
            <p>
                All Thermo instruments are compatible with quantitative approaches such as parallel-reaction monitoring (PRM), data-independent acquisition (DIA), and multiplexing
                using isobaric 
                <a href="https://www.thermofisher.com/ca/en/home/life-science/protein-biology/protein-mass-spectrometry-analysis/protein-quantitation-mass-spectrometry/tandem-mass-tag-systems.html?gclid=CjwKCAjwkJm0BhBxEiwAwT1AXNASmPL8oNXhmoXVMea9yzHRUs9PyTY3hJN3xyJkvDtESm41BJFgOBoCesYQAvD_BwE&ef_id=CjwKCAjwkJm0BhBxEiwAwT1AXNASmPL8oNXhmoXVMea9yzHRUs9PyTY3hJN3xyJkvDtESm41BJFgOBoCesYQAvD_BwE:G:s&s_kwcid=AL!3652!3!597124685923!e!!g!!tmt%20thermo!12451261088!120049585642&cid=bid_pca_msp_r01_co_cp1359_pjt0000_bid00000_0se_gaw_bt_pur_con&gad_source=1">
                    tandem mass tags
                </a>
                (TMT).  
            </p>
            <h5>
                <a href="https://www.thermofisher.com/ca/en/home/industrial/mass-spectrometry/liquid-chromatography-mass-spectrometry-lc-ms/lc-ms-systems/orbitrap-lc-ms/orbitrap-astral-mass-spectrometer.html"><b>Orbitrap Astral</b></a> 
            </h5>
            <p>
                The Orbitrap Astral excels when the highest degree of proteome coverage and sensitivity is required.  It enables the identification of 8000 proteins from a single
                sample with a throughput of 80 samples per day.   
            </p>
            <h5>
                <a href="https://www.thermofisher.com/order/catalog/product/FETD2-10002?SID=srch-srp-FETD2-10002"><b>Fusion Tribrid Lumos</b></a> 
            </h5>
            <p>
                The Fusion Tribrid Lumos offers a variety of workflows, including confident identification of complex post-translational modifications and the ability to tease apart 
                complex cross-linked mixtures. The Lumos’ capabilities are supported by electron transfer dissociation (ETD) and MSn fragmentation. 
            </p>
                </div>
                <div class="col-md-4 col-sm-12">
                    <div>
                        <img src="img/Orbitrap_Astral.jpg" alt="Orbitrap Astral" width="100%">
                    </div>
                </div>
            </div>
                <hr>
            <h3>
                Bruker 
            </h3>
            <p>
                Bruker mass spectrometers coupled to Evosep Ones (HPLCs) enable robust and fast acquisition of samples by utilizing trapped ion mobility technology coupled to parallel
                accumulation serial fragmentation (PASEF). 
            </p>
            <h5>
                <a href="https://www.bruker.com/de/products-and-solutions/mass-spectrometry/timstof/timstof-pro-2.html"><b>timsTOF Pro 2</b></a> 
            </h5>
            <p>
                The timsTOF Pro 2 is our workhorse for standard workflows such as protein-protein interaction and global proteome studies, enabling the identification of 6000 proteins
                per sample at a rate of 30 samples per day. 
            </p>
            <h5>
                <a href="https://www.bruker.com/en/products-and-solutions/mass-spectrometry/timstof/timstof-scp.html"><b>timsTOF SCP</b></a> 
            </h5>
            <p>
                The timsTOF SCP is a single cell proteomics solution, enabling detection of 4000 proteins from a single-cell level sample (250 pg) and can process 40 samples per day. 
            </p>
                <hr>
            <h3>
                Sciex
            </h3>
            <h5>
                TripleTOF 6600 
            </h5>
            <p>
                The Sciex TripleTOF 6600 which is geared towards the identification of protein complexes and protein-protein interactions. 
            </p>
                <hr>
        </div>

    </div>

    <div id="second-section"></div>
        <!-- Header 2 -->
        <div class="stretch-header pt-2 mt-3">
            <h1 class>
                Pricing
            </h1>
        </div>
        <!-- Header 2 -->
         <div class="container mt-3">
            <p>
                These prices are for academic laboratories and are guidelines only. Interested users should contact Cassandra Wong (cwong@lunenfeld.ca) to discuss their project and receive a quotation (including volume discounts).
            </p>
            <hr>

            <span>
                <h5>
                    Sample Preparation Costs (per sample, minimum of four samples)
                </h5>
            </span>

            <!-- Table 1 -->
            <table class="table tabled-striped table-bordered table-responsive">
                <thead class="table-dark">
                    <tr>
                        <th>
                            Service
                        </th>
                        <th>
                            Internal Academic Cost
                        </th>
                        <th>
                            External Academic Cost
                        </th>
                    </tr>
                </thead>
                <tbody>
                    <tr>
                        <td>
                            Affinity Purification: FLAG or other tag 
                        </td>
                        <td>
                            $ 120
                        </td>
                        <td>
                            $ 130
                        </td>
                    </tr>
                    <tr>
                        <td>
                            Affinity Purification: Proximity-dependent biotinylation 
                        </td>
                        <td>
                            $ 135
                        </td>
                        <td>
                            $ 150
                        </td>
                    </tr>
                    <tr>
                        <td>
                            Lysate preparation and trypsin digest 
                        </td>
                        <td>
                            $ 120
                        </td>
                        <td>
                            $ 140
                        </td>
                    </tr>
                    <tr>
                        <td>
                            Phosphopeptide enrichment 
                        </td>
                        <td>
                            $ 180
                        </td>
                        <td>
                            $ 200
                        </td>
                    </tr>
                    <tr>
                        <td>
                            In-solution trypsin digest 
                        </td>
                        <td>
                            $ 50
                        </td>
                        <td>
                            $ 55
                        </td>
                    </tr>
                </tbody>
            </table>
            <hr>
            <!-- Table 1 -->

            <span>
                <h5>
                    Mass Spectrometry Costs per Sample
                </h5>
            </span>

            <!-- Table 2 -->
            <table class="table tabled-striped table-bordered table-responsive">
                <thead class="table-dark">
                    <tr>
                        <th>
                            Service
                        </th>
                        <th>
                            Inernal Academic Cost
                        </th>
                        <th>
                            External Academic Cost
                        </th>
                    </tr>
                </thead>
                <tbody>
                    <tr>
                        <td>
                            Identification (DDA) or Quantification (DIA) - AP-MS or BioID  
                        </td>
                        <td>
                            $ 110
                        </td>
                        <td>
                            $ 120
                        </td>
                    </tr>
                    <tr>
                        <td>
                            Quantification (DIA) - cell lysate or cell fractionation  
                        </td>
                        <td>
                            Starting $ 135
                        </td>
                        <td>
                            Starting $ 150
                        </td>
                    </tr>
                    <tr>
                        <td>
                            Quantification - DDA with TMT
                        </td>
                        <td>
                            Starting $ 360
                        </td>
                        <td>
                            Starting $ 380 
                        </td>
                    </tr>
                </tbody>
            </table>
            <p>
                Customized mass spectrometry analyses are also available. 
             </p>
             <hr>
            <!-- Table 2 -->

            <p>
                All services include initial consultation and project planning with a proteomics manager, database searching including SAINTexpress for protein interactions,
                full report and data consultation. For additional data analysis, the fee is $90/hour (internal) and $99/hour (external). 
            </p>
            <hr>
         </div>   
    </div>

    <div id="fourth-section">

        <!-- Header 4 -->
        <div class="stretch-header pt-2 mt-3">
            <h1 class>
                Protocols
            </h1>
        </div>
        <!-- Header 4 -->
         

        <div class="container">
            <div id="protocol" class="mt-4">
                <h3>
                    Protocols
                </h3>
                <table class="table table-striped table-bordered table-responsive">
                    <thead class="table-dark">
                        <tr>
                            <th>
                                Name
                            </th>
                            <th>
                                PDF
                            </th>
                        </tr>
                    </thead>
                    <tbody>
                        <tr>
                            <td>
                                NBCC PR1 In-gel digest
                            </td>
                            <td>
                                <a href="assets/PDF/NBCC PR1 Gel staining and in-gel digestion.pdf"><i class="bi bi-file-earmark-pdf"></i></a>
                            </td>
                        </tr>
                        <tr>
                            <td>
                                NBCC PR2 FLAG-IP
                            </td>
                            <td>
                                <a href="assets/PDF/NBCC PR2 FLAG IP.pdf"><i class="bi bi-file-earmark-pdf"></i></a>
                            </td>
                        </tr>
                        <tr>
                            <td>
                                NBCC PR3 BioID Flp-In T-REx cell prep
                            </td>
                            <td>
                                <a href="assets/PDF/NBCC PR3 BioID Flp-IN T-REx Cell Prep.pdf"><i class="bi bi-file-earmark-pdf"></i></a>
                            </td>
                        </tr>
                        <tr>
                            <td>
                                NBCC PR4 BioID lentiviral cell prep
                            </td>
                            <td>
                                <a href="assets/PDF/NBCC PR4 BioID Lentiviral Cell Prep.pdf"><i class="bi bi-file-earmark-pdf"></i></a>
                            </td>
                        </tr>
                        <tr>
                            <td>
                                NBCC PR5 BioID affinity purification
                            </td>
                            <td>
                                <a href="assets/PDF/NBCC PR5 BioID affinity purification and digestion.pdf"><i class="bi bi-file-earmark-pdf"></i></a>
                            </td>
                        </tr>
                    </tbody>
                </table>
                <p>
                    <b>Note: For miniTurbo, TurboID, and ultraID, recommended biotin labeling time is 1 hour.</b>
                </p>

            </div>
        </div>

        <!-- Header 4 -->
        <div class="stretch-header pt-2 mt-3">
            <h1 class>
                Policies
            </h1>
        </div>
        <!-- Header 4 -->
         
         <div class="container mt-3">
            <h3>
                General Policies
            </h3>
                <p>
                    <li style="font-size: 20px;">
                        Hours of Operation are 8 am to 4 pm Monday to Friday.  Please contact Cassandra Wong (cwong@lunenfeld.ca) to discuss your project before dropping off any samples.  A <a href="assets/PDF/NBCC_service_request_form_2024.xlsx">sample intake</a> sheet must be completed for every submission. 
                    </li>
                </p>
                <p>
                    <li style="font-size: 20px;">
                        Samples should only be shipped on Monday or Tuesday to ensure samples arrive before the weekend. Please ensure there is sufficient dry ice to account for any shipping delays. Ship samples to “Cassandra Wong, Mount Sinai Hospital, 600 University Ave, Room 968, Toronto, ON, M5G 1X5” and send tracking information to Cassandra Wong (cwong@lunenfeld.ca).
                    </li>
                </p>
            <h3>
                Acknowledgement
             </h3>
            <p>
                Preferred acknowledgment for the Network Biology Collaborative Centre Proteomics Facility:  
            </p>
            <p>
                <b>The authors wish to thank [staff name] of the Network Biology Collaborative Centre Proteomics Facility (RRID: SCR_025375) at the Lunenfeld-Tanenbaum Research Institute 
                for [service].  The facility is supported by the Canada Foundation for Innovation and the Ontario Government.</b> 
            </p>
            <p>
                Click 
                <a href="publications.html">
                    here
                </a>
                for our publication policy and preferred acknowledgement for other NBCC Facilities.  
            </p>
         </div>

    </div>

    <!--Return to Top-->
    <a href="#" class="back-to-top">
        <i class="bi bi-arrow-up-circle-fill"></i>
    </a>
    <!--Return to Top-->

    

    <!--Footer Start-->
    <div class="container-fluid mt-5 footer">
        <hr>
          <div class="row mb-4">
              <div class="col-md-4 col-sm-4 col-xs-4">
                  <div class="footer-text">
                      <div class="">
                          <h1 style="color: #FBB040;">
                            NBCC
                          </h3>
                          <h5>
                            Network Biology Collaborative Centre
                          </h5>
                          <p>
                            Sinai Health, Lunenfeld-Tanenbaum Research Institute
                            <br>
                            600 University Avenue, Room 970
                            <br>
                            Toronto, Ontario
                            Canada
                            M5G 1X5
                          </p>
                      </div>
                      <div class="social mt-2 mb-3"> <i class="fa fa-facebook-official fa-lg"></i> <i class="fa fa-instagram fa-lg"></i> <i class="fa fa-twitter fa-lg"></i> <i class="fa fa-linkedin-square fa-lg"></i> <i class="fa fa-facebook"></i> </div>
                  </div>
              </div>
              
              <div class="col-md-1 col-sm-1 col-xs-1"></div>
              <div class="vr p-0 d-none d-sm-block"></div>
              <div class="d-md-none"><hr></div>
              <div class="col-md-2 col-sm-2 col-xs-2">
                  <h5 class="heading">Facilities</h5>
                  <ul>
                      <li>
                        <a href="proteomics.html">Proteomics</a>
                      </li>
                      <li>
                        <a href="Next-Generation_Sequencing.html">Next-Generation Sequencing</a>
                      </li>
                      <li>
                        <a href="High-Throughput_Screening.html">High-Throughput Screening</a>
                      </li>
                      <li>
                        <a href="High-Content_Screening.html">High-Content Screening</a>
                      </li>
                      <li>
                        <a href="Advanced_Imaging.html">Advanced Imaging</a>
                      </li>
                      <li>
                        <a href="Flow_Cytometry.html">Flow Cytometry</a>
                    </li>
                  </ul>
              </div>
              <div class="vr p-0 d-none d-sm-block"></div>
              <div class="d-md-none"><hr></div>
              <div class="col-md-2 col-sm-2 col-xs-2">
                  <h5 class="heading">More</h5>
                  <ul class="card-text">
                    <li>
                        <a href="resources.html">Resources</a>
                    </li>
                    <li>
                        <a href="publications.html">Publications</a></li>
                    <li>
                        <a href="news.html">News</a>
                    </li>
                    <li>
                        <a href="about.html">About</a>
                    </li>
                    <li>
                        <a href="team.html">Team</a>
                    </li>
                    <li>
                        <a href="contact.html">Contact</a>
                    </li>
                  </ul>
              </div>
              <div class="vr p-0 d-none d-sm-block"></div>
              <div class="d-md-none"><hr></div>
              <div class="col-md-2 col-sm-2 col-xs-2">
                  <h5 class="heading">Socials</h5>
                  <ul class="card-text">
                    <li>
                        <a href="https://www.linkedin.com/company/network-biology-collaborative-centre/">LinkedIn</a>
                    </li>
                    <li>
                        <a href="https://x.com/nbcc_ltri">X</a>
                    </li>
                  </ul>
              </div>
              <div class="vr p-0 d-none d-sm-block"></div>
          </div>
          <hr>
          <div class="footer_copyright">
            <p>
                <i class="bi bi-c-circle"></i>
                 2024 Network Biology Collaborative Centre, Lunenfeld-Tanenbaum Research Institute, Sinai Health System
            </p>
        </div>
    </div>
    <!--Footer End-->
    <!-- Bootstrap JS -->
    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.1/dist/js/bootstrap.bundle.min.js"
        integrity="sha384-/bQdsTh/da6pkI1MST/rWKFNjaCP5gBSY4sEBT38Q/9RBh9AH40zEOg7Hlq2THRZ" crossorigin="anonymous">
    </script>
    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.8/dist/umd/popper.min.js"
    integrity="sha384-I7E8VVD/ismYTF4hNIPjVp/Zjvgyol6VFvRkX/vR+Vc4jQkC+hVqc2pM8ODewa9r" crossorigin="anonymous">
    </script>
</body>

</html>